### DISCUSSION TITLE
How can I set the Y-axis in ImPlot so that the top is negative and the bottom is positive?

### DISCUSSION THREAD

**Discussion by RWBYBING** [epezent/implot#660] [Q&A]
I'm trying to reverse the Y-axis in ImPlot, where I want negative values at the top and positive values at the bottom. What is the best way to achieve this?

**Comment by brenocq** [comment]
Hi @RWBYBING, did you try using the flag `ImPlotAxisFlags_Invert` with the Y-axis?
